framework/components/ecl-site-headers/_site-headers.scss

```diff
@@ -5,18 +5,18 @@
 
 .ecl-site-header {
   margin: 0;
-  padding: map-get($ecl-spacing, 's') 0;
   width: 100%;
 }
 
-.ecl-site-header__container {
+.ecl-site-header__banner {
   align-items: center;
   display: flex;
   flex-wrap: wrap;
   justify-content: center;
+  padding: map-get($ecl-spacing, 's') 0;
```

Comment:
I just moved it from .ecl-site-header to .ecl-site-header__banner, as now the site header also contains other elements (and should not have padding)